framework/SupervisedLearning/ARMA.py

```diff
@@ -188,7 +237,7 @@ def __trainLocal__(self,featureVals,targetVals):
     self.raiseADebug('Training...')
     # DEBUG FILE -> uncomment lines with this file in it to get series information.  This should be made available
     #    through a RomTrainer SolutionExport or something similar, or perhaps just an Output DataObject, in the future.
-    # debugfile = open('debugg_varma.csv','w')
+    #debugfile = open('debugg_varma.csv','w')
```

Comment:
remove?

Reply:
You'll see there's a comment immediately above this one that explains why it is left in place.  This applies to all similar comments regarding this a a future implementation.